sdk/identity/azure-identity/azure/identity/aio/_credentials/azure_cli.py

```diff
@@ -100,12 +100,13 @@ async def _run_command(command: str) -> str:
         proc = await asyncio.create_subprocess_exec(
             *args,
             stdout=asyncio.subprocess.PIPE,
-            stderr=asyncio.subprocess.STDOUT,
+            stderr=asyncio.subprocess.PIPE,
             cwd=working_directory,
             env=dict(os.environ, AZURE_CORE_NO_COLOR="true")
         )
-        stdout, _ = await asyncio.wait_for(proc.communicate(), 10)
-        output = stdout.decode()
+        stdout_b, stderr_b = await asyncio.wait_for(proc.communicate(), 10)
+        output = stdout_b.decode()
```

Comment:
Why we need this?

Reply:
I see you have

output = stdout_b.decode()
stderr = stderr_b.decode()

and seems like now we only use stderr.

Is there any reason to keep output = stdout_b.decode()?

Reply:
output is the actual return value on line 119 :)

Reply:
Why can't we return stderr? :)

They are same, no?

Reply:
From what I understand, stdout contains the actual access token json output returned by the az cli, while stderr will naturally have the error messages and sometimes warnings as in the tagged issue's case.